src/commands/containers/aciContainerActionWarning.ts

```diff
@@ -0,0 +1,36 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { DialogResponses, IActionContext } from 'vscode-azureextensionui';
+import { ext } from '../../extensionVariables';
+import { localize } from '../../localize';
+import { getComposeProjectName, NonComposeGroupName } from '../../tree/containers/ContainersTreeItem';
+import { ContainerTreeItem } from '../../tree/containers/ContainerTreeItem';
+
+export async function aciContainerActionWarning(context: IActionContext, nodes: ContainerTreeItem[]): Promise<string[]> {

```

Comment:
Nit: the name of this function could be better, as it is not just about showing a warning to the user; it also returns all containers affected by the operation. Consider `confirmAllAffectedContainers()` or something like that

Reply:
Agree, I never liked the name of this method. Good suggestion.